hw/dv/dpi/usbdpi/usbdpi.c

```diff
@@ -93,8 +97,11 @@ void usbdpi_device_to_host(void *ctx_void, const svBitVecVal *usb_d2p) {
   char obuf[MAX_OBUF];
 
   char raw_str[D2P_BITS + 1];
-  for (int i = 0; i < 5; i++) {
-    raw_str[5 - i - 1] = !!(d2p & (1 << i)) + '0';
+  {
```

Comment:
Hmm. The reason I changed this, is that the original version does not compile in our Incisive version. It compiles the .c file as an ANSI C file, while Verilator seems to compile it as a C++ file... I'm not sure how to solve this, but I would like to avoid having two separate versions of the file.